src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -70,6 +70,9 @@
         - name: --enable-aad
           type: bool
           short-summary: Enable managed AAD feature for cluster.
+        - name: --enable-azure-rbac
```

Comment:
could you pls use `enable-rbac` as param name to consistent with existing commands?

Reply:
enable-rbac has a different functionality then enable-azure-rbac though and in the api we have added enableAzureRBAC as the param. Also, enable-azure-rbac will only be for aad v2 clusters at the time of creation.

Reply:
thanks. what's difference between `enable-rbac` and `enable-azure-rbac`?

Reply:
enable-rbac enables K8s RBAC mode for authorization where as enable-azure-rbac will enable webhook mode in K8s for authorization, the webhook server that we have deployed using guard which will receive the authorization requests from K8s and convert them into appropriate checkaccess calls to send to ARM and then return the result whether the user is authorized to perform the selected action or not